lib/moonshot/ssh_config.rb

```diff
@@ -2,10 +2,12 @@ module Moonshot
   class SSHConfig
     attr_accessor :ssh_identity_file
     attr_accessor :ssh_user
+    attr_accessor :ssh_optional_parameters
 
     def initialize
       @ssh_identity_file = ENV['MOONSHOT_SSH_KEY_FILE']
       @ssh_user = ENV['MOONSHOT_SSH_USER']
+      @ssh_optional_parameters = []
```

Comment:
Maybe this can just be a string, and we can define a default environment variable for it MOONSHOT_SSH_OPTIONS?

Reply:
And does it have to be an array, can it just be a string, for example "-v -o StrictHostKeyChecking=no". Might help with easy configuration from environment variable or directly passing the string.

Reply:
That's what we discussed last time. It can be a string and yes, we can convert it to an ENV variable.

Reply:
I converted it to ENV. I can see in `/spec/moonshot/ssh_spec.rb` the value is being used now, and the Moonshot core ssh is getting the new value.

Reply:
The valid solution to this is I removed initialization from [here](https://github.com/acquia/moonshot/pull/270/files#diff-02d46b8526660e3a96b20d4108ece8e462599ab08f0dac29f15845a2725a458fR7) for the `attr_accessor` and just inject the value through [cdb-worker Moonfile.rb](https://github.com/acquia/cloud-database-worker/pull/1020/files#diff-9a6c7ec0145dc4264f61b3a8be25fb874d9bd82b1fa35a53a7eb1a194ff2747bR117)